Lib/test/test_capi/test_bytes.py

```diff
@@ -53,6 +53,8 @@ def test_fromstringandsize(self):
         self.assertEqual(fromstringandsize(b'abc'), b'abc')
         self.assertEqual(fromstringandsize(b'abc', 2), b'ab')
         self.assertEqual(fromstringandsize(b'abc\0def'), b'abc\0def')
+        self.assertEqual(fromstringandsize(b'a'), b'a')
+        self.assertEqual(fromstringandsize(b'a', 1), b'a')
```

Comment:
That seems to be redundant, isn't?

```suggestion
```

Reply:
In the current `bytes_fromstringandsize` implementation there are two paths where `size` argument can be calculated. I prefer to cover both of them.